{"name": "test_login_invalid_credentials", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to be enabled\nActual value: disabled \nCall log:\n  - LocatorAssertions.to_be_enabled with timeout 5000ms\n  - waiting for locator(\"button[type='submit']:has-text('Log in')\")\n    9 × locator resolved to <button type=\"submit\" aria-disabled=\"true\" data-test=\"login-submit\" class=\"login-page-new__main-form-button\">…</button>\n      - unexpected value \"disabled\"", "trace": "page = <Page url='https://app.clickup.com/login'>\n\n    @allure.feature(\"UI: Авторизация ClickUp, негативный сценарий\")\n    @allure.description(\"Проверяем, что пользователь не может войти с неправильным логином или паролем\")\n    def test_login_invalid_credentials(page):\n        login_page = LoginPage(page)\n    \n        invalid_email = \"rtytr@yfgf.com\"\n        invalid_password = \"wrongpass\"\n    \n>       login_page.login_with(invalid_email, invalid_password)\n\ntest_login_page.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\pages\\login_page.py:32: in login_with\n    self.wait_for_selector_and_click(self.LOGIN_BUTTON_SELECTOR)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.login_page.LoginPage object at 0x00000121FF641510>\nselector = \"button[type='submit']:has-text('Log in')\"\n\n    @allure.step(\"Ждём, пока кнопка станет кликабельной, затем кликаем\")\n    def wait_for_selector_and_click(self, selector):\n        locator = self.page.locator(selector)\n    \n        # Явно ждём, пока элемент будет видимым\n        locator.wait_for(state=\"visible\", timeout=10000)\n    \n        # Добавим ожидание enabled\n>       expect(locator).to_be_enabled(timeout=5000)\nE       AssertionError: Locator expected to be enabled\nE       Actual value: disabled \nE       Call log:\nE         - LocatorAssertions.to_be_enabled with timeout 5000ms\nE         - waiting for locator(\"button[type='submit']:has-text('Log in')\")\nE           9 × locator resolved to <button type=\"submit\" aria-disabled=\"true\" data-test=\"login-submit\" class=\"login-page-new__main-form-button\">…</button>\nE             - unexpected value \"disabled\"\n\n..\\..\\pages\\base_page.py:40: AssertionError"}, "description": "Проверяем, что пользователь не может войти с неправильным логином или паролем", "steps": [{"name": "Логинимся с email='rtytr@yfgf.com'", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to be enabled\nActual value: disabled \nCall log:\n  - LocatorAssertions.to_be_enabled with timeout 5000ms\n  - waiting for locator(\"button[type='submit']:has-text('Log in')\")\n    9 × locator resolved to <button type=\"submit\" aria-disabled=\"true\" data-test=\"login-submit\" class=\"login-page-new__main-form-button\">…</button>\n      - unexpected value \"disabled\"\n\n", "trace": "  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\pages\\login_page.py\", line 32, in login_with\n    self.wait_for_selector_and_click(self.LOGIN_BUTTON_SELECTOR)\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\pages\\base_page.py\", line 40, in wait_for_selector_and_click\n    expect(locator).to_be_enabled(timeout=5000)\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 20258, in to_be_enabled\n    self._sync(self._impl_obj.to_be_enabled(enabled=enabled, timeout=timeout))\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 673, in to_be_enabled\n    await self._expect_impl(\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 74, in _expect_impl\n    raise AssertionError(\n"}, "steps": [{"name": "Переход по адресу BASE_URL + endpoint", "status": "passed", "steps": [{"name": "Формируем полный URL из BASE и endpoint", "status": "passed", "start": 1749832135962, "stop": 1749832135962}], "start": 1749832135962, "stop": 1749832153403}, {"name": "Ждем селектор и печатаем значением", "status": "passed", "parameters": [{"name": "selector", "value": "'#login-email-input'"}, {"name": "text", "value": "'rtytr@yfgf.com'"}, {"name": "delay", "value": "100"}], "start": 1749832153403, "stop": 1749832155189}, {"name": "Ждем селектор и печатаем значением", "status": "passed", "parameters": [{"name": "selector", "value": "'#login-password-input'"}, {"name": "text", "value": "'wrongpass'"}, {"name": "delay", "value": "100"}], "start": 1749832155189, "stop": 1749832156309}, {"name": "Ждём, пока кнопка станет кликабельной, затем кликаем", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to be enabled\nActual value: disabled \nCall log:\n  - LocatorAssertions.to_be_enabled with timeout 5000ms\n  - waiting for locator(\"button[type='submit']:has-text('Log in')\")\n    9 × locator resolved to <button type=\"submit\" aria-disabled=\"true\" data-test=\"login-submit\" class=\"login-page-new__main-form-button\">…</button>\n      - unexpected value \"disabled\"\n\n", "trace": "  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\pages\\base_page.py\", line 40, in wait_for_selector_and_click\n    expect(locator).to_be_enabled(timeout=5000)\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 20258, in to_be_enabled\n    self._sync(self._impl_obj.to_be_enabled(enabled=enabled, timeout=timeout))\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 673, in to_be_enabled\n    await self._expect_impl(\n  File \"C:\\Users\\A\\PycharmProjects\\Project_Сlickup_tests\\clickup_tests\\venv\\lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 74, in _expect_impl\n    raise AssertionError(\n"}, "parameters": [{"name": "selector", "value": "'button[type='submit']:has-text('Log in')'"}], "start": 1749832156309, "stop": 1749832161361}], "parameters": [{"name": "email", "value": "'rtytr@yfgf.com'"}, {"name": "password", "value": "'wrongpass'"}], "start": 1749832135962, "stop": 1749832161362}], "start": 1749832135961, "stop": 1749832161362, "uuid": "a23f7ad1-d9be-4e47-8105-8385bbc9a0bc", "historyId": "071f92c4f67a6f129ce94efc090a7ec0", "testCaseId": "071f92c4f67a6f129ce94efc090a7ec0", "fullName": "tests.ui.test_login_page#test_login_invalid_credentials", "labels": [{"name": "feature", "value": "UI: Авторизация ClickUp, негативный сценарий"}, {"name": "parentSuite", "value": "tests.ui"}, {"name": "suite", "value": "test_login_page"}, {"name": "host", "value": "A1"}, {"name": "thread", "value": "47520-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.ui.test_login_page"}]}